packages/compose/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@wordpress/compose",
+	"version": "1.0.0-beta.0",
+	"description": "WordPress Higher Order components",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"React",
+		"hoc"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/compose/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"dependencies": {
+		"@wordpress/element": "file:../element",
+		"@wordpress/is-shallow-equal": "file:packages/is-shallow-equal",
+		"lodash": "4.17.10"
```

Comment:
Can we use range for all dependencies, e.x.:
`"lodash": "^4.17.10"`
We use that in all other packages.

Reply:
the same applies do `devDependencies` :)